README.md

```diff
@@ -16,30 +16,29 @@ This tool is released for users seeking to deploy their own copies of the New Re
 The AWS cli must be configured, please refer to its [documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html).
 
 ## Publishing Layers:
-
-Run the following in your shell:
+To publish the layer, modify the runtime according to the options provided in the `.publish-layers.sh` script. Then, run the following command in your shell:
 
 ```
 cd python
-./publish-layers.sh
+./publish-layers.sh python3.10 
```

Comment:
As we have python3.12 also in the script file. Can we change this to latest available or 3.11 which is stable state ?

Reply:
Update to latest